projects/plugins/boost/app/modules/Modules_Setup.php

```diff
@@ -38,6 +38,21 @@ public function have_enabled_modules() {
 		return false;
 	}
 
+	/**
+	 * Get modules that are currently active and optimizing the site.
+	 *
+	 * @return string[] Slugs of optimization modules that are currently active and serving.
+	 */
+	public function get_active_optimization_modules() {
+		$working_modules = array();
+		foreach ( $this->available_modules as $slug => $module ) {
+			if ( $module->is_optimizing() ) {
+				$working_modules[] = $slug;
+			}
+		}
+		return $working_modules;
+	}
```

Comment:
`get_active_optimization_modules` returns the ones which are `optimizing`. So ... 

If Critical CSS is enabled, but has not generated Critical CSS successfully, it is active and it is an optimization module but this function won't return it. 

`get_optimizing_modules()` perhaps? Even that's a bit complex.

I'm not sure what to call this, but I think the current name is a bit misleading.